controllers/storagecluster/storageclasses.go

```diff
@@ -413,6 +413,20 @@ func newEncryptedCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageC
 	return encryptedStorageClassConfig
 }
 
+// newEncryptedCephFileSystemStorageClassConfiguration generates configuration options for an encrypted Ceph File System StorageClass.
+// when user has asked for PV encryption during deployment.
+func newEncryptedCephFileSystemStorageClassConfiguration(initData *ocsv1.StorageCluster, serviceName string) StorageClassConfiguration {
+	allowVolumeExpansion := true
+	encryptedStorageClassConfig := newCephFilesystemStorageClassConfiguration(initData)
+	encryptedStorageClassConfig.storageClass.ObjectMeta.Name = generateNameForEncryptedCephFileSystemSC(initData)
+	// adding a annotation to support smart cloning across namespace for encrypted volume
+	encryptedStorageClassConfig.storageClass.ObjectMeta.Annotations["cdi.kubevirt.io/clone-strategy"] = "copy"
```

Comment:
Should confirm with kubevirt team once if they want to support cephfs volumes for this as well before adding the annotation.

Reply:
@nixpanic Can you pls confirm this if you have an idea?

Reply:
i dont think we use cephfs for kubevirt. do we have any cephfs storageclass with this annotation in ocs-operator?

Reply:
No only encrypted rbd is the one where it was added.

Reply:
I removed the annotation.